gatsby-ssr.js

```diff
@@ -23,13 +22,38 @@ export const onRenderBody = ({ setHeadComponents }) => {
   }`;
 
   return setHeadComponents([
-    <meta key="og:image" property="og:image" content={ogimage} />,
+    <meta
+      key="description"
+      name="description"
+      content="Carbon is IBM’s open source design system for products and digital experiences. With the IBM Design Language as its foundation, the system consists of working code, design tools and resources, human interface guidelines, and a vibrant community of contributors."
+    />,
+    <meta key="image" name="image" content={ogimage} />,
+    <meta
+      key="og:url"
+      property="og:url"
+      content="https://www.carbondesignsystem.com"
+    />,
+    <meta key="og:title" property="og:title" content="Carbon Design System" />,
+    <meta
+      key="og:description"
+      property="og:description"
+      content="Carbon is IBM’s open source design system for products and digital experiences. With the IBM Design Language as its foundation, the system consists of working code, design tools and resources, human interface guidelines, and a vibrant community of contributors."
+    />,
+    <meta
+      key="og:image"
+      property="og:image"
+      content={`https://www.carbondesignsystem.com${ogimage}`}
```

Comment:
This URL on the deploy preview ends up pointing to a 404:

https://www.carbondesignsystem.com/static/ogimage-9c8ed31c024a9400f9d212142dce4574.jpg

Reply:
Super weird. I was able to get this going with the REiD site using the same code. Do you think it might be a jpg vs. png thing?